WeakAuras/Profiling.lua

```diff
@@ -360,3 +366,160 @@ function WeakAuras.PrintProfile()
   end
   popup:Show()
 end
+
+RealTimeProfilingWindow = CreateFrame("Frame", nil, UIParent)
+RealTimeProfilingWindow.width = 400
+RealTimeProfilingWindow.height = 300
+RealTimeProfilingWindow.barHeight = 20
+RealTimeProfilingWindow.titleHeight = 15
+RealTimeProfilingWindow.bars = {}
+RealTimeProfilingWindow:SetMovable(true)
+
+local texture = "Interface\\DialogFrame\\UI-DialogBox-Background"
+function RealTimeProfilingWindow:AddBar(num, name, map, total)
```

Comment:
Have you considered using AceGUI instead of raw frame manip?

Reply:
AceGUI scares me but i'll try, it may help later for adding new buttons